userservices/client/standard_test.go

```diff
@@ -655,9 +643,7 @@ var _ = Describe("Standard", func() {
 					})
 
 					It("returns an error", func() {
-						permissions, err := standard.GetUserPermissions(context, "request-user-id", "target-user-id")
-						Expect(err).ToNot(HaveOccurred())
-						Expect(permissions).To(Equal(client.Permissions{
+						Expect(standard.GetUserPermissions(context, "request-user-id", "target-user-id")).To(Equal(client.Permissions{
```

Comment:
Yea, I could, but I've tried to avoid test constants. In many cases constants in tests just obfuscate what the test is trying to accomplish. That is, I've tried to make tests as self-contained as possible. So, read just the code within the `It` block and any related `BeforeEach` blocks and it fully identifies the test. Using test constants would make one more place to look for the meaning of the test.

So, no strong reason, but just the way I've done things.